### ISSUE TITLE
Report a problem: colour contrast fails for a couple of items

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#100]
URL: All - eg. https://hcdev.openplus.ca/en/
User Agent: Chrome, Firefox
Language: Both

Issue Description: There are a couple of colour contrast issues with "Report a problem or mistake on this page" section. 

1. Under “Report a problem or mistake on this page”, the <textarea> form field fails contrast ratio against it adjacent colour when focused. Border for the field at focused state = #E09600; Adjacent colour = #F5F5F5

![image](https://user-images.githubusercontent.com/14280062/136585890-f3bf8009-d0dc-4aee-a101-2cd91c848c88.png)

2. After submitting the “Report a problem”, a success message is displayed containing a “x” button to close it. The “x” fails colour contrast. Foreground = #CCCCCC; Background = #FFFFFF

![image](https://user-images.githubusercontent.com/14280062/136585752-67135b99-bbbb-4524-90dc-3a27cda668d0.png)

Expected results: 

1. Current contrast ratio is 2.26:1 while it should be minimum 3:1.
2. Current contrast ratio is 1.61:1 while it should be minimum 3:1.

**Comment by echouin** [comment]
From OP: ﻿Fixed and will be part of next commit

**Comment by ndolter** [comment]
Validated that the color contrast now passes.
